db/migrate/20160915100611_add_openshift_data_to_container_image.rb

```diff
@@ -0,0 +1,16 @@
+class AddOpenshiftDataToContainerImage < ActiveRecord::Migration[5.0]
+  def change
+    add_column  :container_images, :architecture, :string
+    add_column  :container_images, :author, :string
+    add_column  :container_images, :command, :string
```

Comment:
@simon3z I think it makes sense to save as a string. It might be delivered from openshift as a list of strings (each word separated as an element). In that case do you prefer to keep it a list or to save it as a long string with spaces between the words?